app/Http/Controllers/MyWorkflowsController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Workflow;
+use Auth;
+use App\VerificationComment;
+use App\CommentReply;
+use App\LoadedEvidencioModel;
+use App\Step;
+use App\Result;
+use App\Field;
+
+class MyWorkflowsController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+        $this->middleware('can:view-designer');
+    }
+
+    public function index()
+    {
+
+        $user_id = Auth::user()->id;
+        $workflows = Workflow::where('author_id', '=', $user_id)->get();
+
+        return view('myworkflows', compact('workflows'));
+    }
+
+    public function deleteWorkflow($id)
+    {
```

Comment:
This logic should not be present in your controllers. It should be registered in your model as a delete callback. This happens in a lot of other locations as well.

Reply:
I have done this for the other cases (Designer(Load/Save)Controller), though we will have to test it to be sure it doesn't have bugs.